src/NovelRT/Windowing/Glfw/GlfwWindowingDevice.cpp

```diff
@@ -49,7 +49,10 @@ namespace NovelRT::Windowing::Glfw
             throw Exceptions::InitialisationFailureException("GLFW3 failed to initialise.", std::string(output));
         }
 
-        glfwSetWindowAttrib(window, GLFW_VISIBLE, GLFW_TRUE);
+        #ifndef __APPLE__
+            glfwSetWindowAttrib(window, GLFW_VISIBLE, GLFW_TRUE);
```

Comment:
Indentation looks a bit scuffed here. Clang-format will probably fix this. I'd address the other change first though otherwise you'll be running clang-format twice.